Tasks/DownloadFileShareArtifactsV0/icon.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is this same icon as DownloadBuildArtifacts task? do we need a separate icon for this task?

Reply:
We should get a new icon and have a corresponding counterpart for upload